### PR TITLE
create a pages with product card

### PR DESCRIPTION
**Pull request by AlexTsimba**
- [DEMO LINK](https://alextsimba.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://alextsimba.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
```
Comment (viktoriia-ovcharenko) [inline src/style.css:148]: use two different files for styles card and stars

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+    <div class="card" data-qa="card">
+
+      <img
```
Comment (viktoriia-ovcharenko) [inline src/index.html:13]: the third point of the checklist

File: src/style.css (hunk 1 above)
Comment (viktoriia-ovcharenko) [inline src/style.css:148]: group styles by type

Comment (AlexTsimba) [reply src/style.css]: looks like you meant the block above
styles are grouped in order

File: src/index.html (hunk 2 above)
Comment (AlexTsimba) [reply src/index.html:15]: redundant spacing fixed

File: src/style.css (hunk 1 above)
Comment (AlexTsimba) [reply src/style.css]: stars block styles moved to a separate file

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,60 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <!-- / Stars block HTML from Stars Layout -->
+
+        <p class="stars-block__reviews-count reset">Reviews: 5</p>
+      </div>
```
Comment (Olexandr6) [inline src/index.html:43]: As we discussed on the Q&A, let's don't leave comments in the production code
```suggestion
      <div class="stars-block">
        <div class="stars">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        
        <p class="stars-block__reviews-count reset">Reviews: 5</p>
      </div>
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,60 @@
...
+
+      <div class="card__price">
+        <p class="card__price-name reset">Price:</p>
+
+        <p class="card__amount reset">$2,199</p>
```
Comment (Olexandr6) [inline src/index.html:48]:
```suggestion
        <p class="card__price-name reset">Price:</p>
        <p class="card__amount reset">$2,199</p>
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
+@import url("./stars.css");
+
+:root {
+  --main-color: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray: #e5e5e5;
+}
```
Comment (Olexandr6) [inline src/style.css:9]: I would create a separate file for the variables - `variables.css`

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+  padding: 16px 16px 0;
+}
+
+.stars-block__reviews-count {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+}
```
Comment (Olexandr6) [inline src/style.css:126]:
```suggestion
  line-height: 14px;
}
```

**Review by Olexandr6 — APPROVED** [review]
Overall - good job! I will approve, but make sure to fix my comments👇

**Pull request closed without merging**